pkg/tsdb/azuremonitor/metrics/url-builder.go

```diff
@@ -18,7 +18,7 @@ type urlBuilder struct {
 	ResourceName        string
 }
 
-func (params *urlBuilder) buildMetricsURLFromLegacyQuery() string {
+func (params *urlBuilder) BuildResourceURIFromLegacyQuery() string {
```

Comment:
any reason this method was capitalized? it will still be accessible from pkg/tsdb/azuremonitor/metrics/azuremonitor-datasource_test.go since they're in the same package.

Reply:
You're right, I was still thinking in javascript 😆

Reply:
Yeah I know it's confusing - I'm constantly mixing things up. :)